yarn.lock

```diff
@@ -0,0 +1,6621 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why both `package-lock.json` and `yarn.lock`? I'd say pick one package manager that you'll use for the project.